fendermint/app/src/cmd/genesis.rs

```diff
@@ -248,7 +251,7 @@ fn into_tendermint(genesis_file: &PathBuf, args: &GenesisIntoTendermintArgs) ->
         // Hopefully leaving this empty will skip validation,
         // otherwise we have to run the genesis in memory here and now.
         app_hash: tendermint::AppHash::default(),
-        app_state: genesis_json,
+        app_state: serde_json::Value::String(app_state),
```

Comment:
I suggest renaming the parse_genesis_app_bytes function to deserialise_genesis_app_bytes for better clarity and consistency. Additionally, it would be beneficial to add a corresponding serialise_genesis_app_bytes function. To keep related functionality together, consider moving both functions to a new module, perhaps named serde, within the genesis package. This

Reply:
I think this might need a more in depth refactor, because the serialization is in `app` but deserialization in interpreter.